runtime/j9vm/jvm.c

```diff
@@ -4967,19 +4967,26 @@ JVM_Accept(jint descriptor, struct sockaddr* address, int* length)
 
 			returnVal = select(descriptor+1, &fdset, 0, 0, &tval);
 		} while(returnVal == 0);
-		retVal = accept(descriptor, address, &socklen);
+
+		do {
+			retVal = accept(descriptor, address, &socklen);
+		} while (errno == EINTR);
 		*length = (int)socklen;
 	}
 #elif defined (WIN32)
 	{
-		SOCKET socketResult = accept(descriptor, address, length);
+		do {
+			SOCKET socketResult = accept(descriptor, address, length);
+		} while (errno == EINTR);
```

Comment:
Windows doesn't have `errno` and `EINTR`. Current code won't compile on Windows.

On Windows, `accept` returns `INVALID_SOCKET` in case of error. ` WSAGetLastError()` will return the error code. 

Check for retrying accept will be like this:
`} while ((INVALID_SOCKET == socketResult) && (WSAEINTR == WSAGetLastError()));`

Windows `accept` documentation: https://msdn.microsoft.com/en-us/library/windows/desktop/ms737526(v=vs.85).aspx